src/uucp_break.ml

```diff
@@ -35,6 +35,38 @@ let sentence u = Array.unsafe_get Low.sentence_of_int (Low.sentence u)
 
 let east_asian_width u = Uucp_rmap.get Uucp_break_data.east_asian_width_map u
 
+let err_ctrl_char e =
+  invalid_arg (Printf.sprintf "0x%02X is a control character" e)
+
+let err_not_uchar e =
+  invalid_arg (Printf.sprintf "U+%04X is not a unicode scalar value." e)
+
+let terminal_width_hint =
+  let gc = Uucp_gc.general_category in
+  function
+  (* C0 (without 0x00) + DELETE + C1 is an error. *)
+  |u when 0 < u && u <= 0x1f || 0x7f <= u && u <= 0x9f -> err_ctrl_char u
```

Comment:
Btw. I still don't really understand the rationale in using `0` for `U+2028` and `U+2029`, but `-1` for U+0085, U+000A and U+000D ?